flow/util/genMassive.py

```diff
@@ -424,8 +424,8 @@ def writeConfigs(CurAttrs, CurChunkNum):
   frun.close()
   
   
-  fcollect = open('./%s_metrics_collect.sh'%ShellName, 'a')
-  CollectName = 'python util/genMetrics.py -x -p %s -d %s -v %s -o metrics_%s/%s.json\n'%(CurPlatform, CurDesign, variantName, ShellName, variantName)
+  fcollect = open('./metrics/%s_metrics_collect.sh'%ShellName, 'a')
+  CollectName = 'python util/genMetrics.py -x -p %s -d %s -v %s -o metrics/metrics_%s/%s.json\n'%(CurPlatform, CurDesign, variantName, ShellName, variantName)
   fcollect.write(CollectName)
   fcollect.close()
```

Comment:
```
with open(...) as fcollect:
    fcollect.write()
```
I think this is safer as it guarantees file is closed

Reply:
Hi Vitor, I applied all your comments to the new commits. Thank you so much for your comments!